src/ocrd/processor/helpers.py

```diff
@@ -374,28 +374,29 @@ def get_cached_processor(parameter: dict, processor_class):
         Otherwise, an instance of the `:py:class:~ocrd.Processor` is returned.
     """
     if processor_class:
-        dict_params = dict(parameter) if parameter else None
-        processor = processor_class(None, parameter=dict_params)
-        processor.setup()
+        processor = processor_class(None, parameter=dict(parameter))
         return processor
     return None
 
 
 def get_processor(
         processor_class,
-        parameter: dict,
+        parameter: Optional[dict],
```

Comment:
Any reason to still have this as a required arg?
```suggestion
        parameter: Optional[dict] = None,
```

Reply:
Not really, no. But then `get_cached_processor` should be changed analogously.